### PR TITLE
List with cards

### PR DESCRIPTION
**Pull request by RealBirdMan91**
User and List have to be linked together

### CODE REVIEW

**Comment by amyria3** [comment]
Authentification seems to work

File: package.json [hunk 1]
```diff
@@ -67,7 +70,7 @@
     "axios": "^1.3.4",
     "babel-loader": "^8.3.0",
     "postcss": "^8.4.21",
-    "prisma": "^4.10.1",
+    "prisma": "4.12.0-dev.34",
```
[the remark this answers is not in the document]
Comment (NoHara42) [reply package.json:73]: in other words, please put this back to the previous version unless there is an explicit reason - dev releases aren't normally stable.

File: src/components/Lists.tsx
Comment (NoHara42) [inline src/components/Lists.tsx:1]: delete this please

File: src/pages/api/createList.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { z, ZodError } from "zod";
 import { prisma } from "./prisma";
 
 /* The point gets user id and should take name, if
```
Comment (NoHara42) [inline src/pages/api/createList.ts:5]: ??

File: src/stories/Card.stories.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { Meta, StoryObj } from "@storybook/react";
-import Card from "../components/Card";
+import Card from "../components/SingleCard";
```
Comment (NoHara42) [inline src/stories/Card.stories.tsx:2]: Card should be SingleCard here right?

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see small changes

**Comment by iameru** [comment]
Please recheck, all changes requested are done, except for storybook which I reverted to show the originial `Card` component

**Comment by NoHara42** [comment]
@iameru 

Issues:
- when you delete a list item you get an error
- when you drag a list item when inspecting on desktop dimensions, you immediately enter the list items
- User name is not displayed instead of the "Hey Beautiful" placeholder
- Draft lists (lists that have just been created) are not greyed out, and have "NEW LIST" as a default value. 
(A user should not have to remove "NEW LIST" to enter their list name.

- See design issues at iPhone SE dimensions 
image

**Pull request merged**